new/src/sections/home/home.scss

```diff
@@ -0,0 +1,68 @@
+.home {
+  position: relative;
+
+  &-left {
+    padding: 5.7vw 8vw;
+    display: flex;
+    flex-direction: column;
+    gap: 50px;
+  }
+
+  &-text {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+
+    &-lg {
+      font-size: 50px;
+      font-weight: 900;
+      line-height: 130%;
+
+      @media (min-width: 1024px) {
+        width: 42vw;
+      }
+
+      @media (max-width: 768px) {
+        font-size: 40px;
+      }
+    }
+
+    &-creative {
+      font-family: "Leckerli One", cursive;
```

Comment:
can add this font from [google fonts](https://fonts.google.com/specimen/Leckerli+One) too so we can check the text looks fine